ui_framework/components/table/table_row_check_box_cell.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { KuiTableRowCell } from './table_row_cell';
+import { KuiTableRowCellLiner } from './table_row_cell_liner';
+
+export const KuiTableRowCheckBoxCell = ({ onChange, isChecked, className, ...rest }) => {
+  const classes = classNames('kuiTableRowCell--checkBox', className);
+  return <KuiTableRowCell className={ classes } {...rest} >
+    <KuiTableRowCellLiner>
+      <input type="checkbox" className="kuiCheckBox" onChange={ onChange } checked={ isChecked } />
```

Comment:
Ahhh, I had no idea we had that rule.  I thought all this time you were just asking because you personally liked it better that way.  Sorry about that! 😸